llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -3155,25 +3166,153 @@ static AbbrevEntryReader getAbbrevEntryReader(ArrayRef<StringRef> AbbrevEntries,
   return AbbrevEntryReader(AbbrevData);
 }
 
-Error DIEVisitor::visitDIERef(BinaryStreamReader &DataReader,
-                              unsigned AbbrevIdx, StringRef DIEData,
-                              ArrayRef<DIEDataRef> &DIEChildrenStack) {
+static std::pair<bool, uint64_t> getNonULEBFormSize(dwarf::Form Form,
+                                                    dwarf::FormParams FP) {
+  switch (Form) {
+  case dwarf::DW_FORM_addr:
+  case dwarf::DW_FORM_ref_addr: {
+    auto FormSize =
+        (Form == dwarf::DW_FORM_addr) ? FP.AddrSize : FP.getRefAddrByteSize();
+    return std::make_pair(false, FormSize);
```

Comment:
you can probably just write 
```suggestion
    return {false, FormSize};
```